src/lib/ical/utils.ts

```diff
@@ -241,9 +241,13 @@ export function makeOutlookCalendarLink(
 
 export function parseLocation(
   rawLocation?: string,
-  defaultLocation = 'Discord',
+  defaultLocation = 'TBD',
   defaultLink = '/discord'
 ): { location: string; meetingLink: string } {
+  if (!rawLocation) {
+    rawLocation = rawLocation.trim();
+  }
```

Comment:
Nit:
```suggestion
  rawLocation = rawLocation?.trim() ?? '';
```
Note: Now that `rawLocation` always asserts a typeof `string` we can remove the `?.` from the other conditionals in this function.

Reply:
Will make a commit with removing the other null operators